test/traversals/dfs.jl

```diff
@@ -26,8 +27,9 @@
 
   @testset "is_cyclic" begin
     for g in testgraphs(path_graph(2))
-      @test @inferred(is_cyclic(g))
+      @test @inferred(!is_cyclic(g)) 
       @test @inferred(!is_cyclic(zero(g)))
+      @test @inferred(is_cyclic((gcyclic)))
```

Comment:
This should probably go outside of the loop since it is a specific graph unrelated to `path_graph(2)`. Check out how `test_graphs` work, it will become clearer

Reply:
Yes, it should be outside the loop